build.sbt

```diff
@@ -3,12 +3,19 @@ import uk.gov.hmrc.DefaultBuildSettings
 
 val appName = "trusts-store"
 
+ThisBuild / scalaVersion := "2.13.13"
+ThisBuild / majorVersion := 0
+
+val excludedPackages = Seq(
+  "<empty>",
+  ".*Routes.*"
+)
+
 lazy val scoverageSettings = {
   import scoverage.ScoverageKeys
   Seq(
-    ScoverageKeys.coverageExcludedPackages := "<empty>;.*Reverse.*;.*.Routes.*;prod.*;testOnlyDoNotUseInProd.*;testOnlyDoNotUseInAppConf.*;" +
-      ".*BuildInfo.*;app.*;prod.*;config.*",
-    ScoverageKeys.coverageMinimumStmtTotal := 95,
+    ScoverageKeys.coverageExcludedFiles := excludedPackages.mkString(";"),
+    ScoverageKeys.coverageMinimumStmtTotal := 97,
```

Comment:
Is there a reason this was increased?

Reply:
it is reflecting real number now, maybe it increased/decreased after upgrade but this is real value now